### PR TITLE
Added missing current balance in output json and TDDs

### PR DESCRIPTION
**Pull request by LBHRShetty**
Fixed the error of missing current balance in output json
Fixed missing current balance ibn example 1 and example 2 of tests

### CODE REVIEW

File: LBHTenancyAPITest/Test/Controllers/GetAllTenancyDetailsForGivenTenancyRefTest.cs [hunk 1]
```diff
@@ -181,6 +181,7 @@ public async Task WhenGivenATenancyRef_TenancyDetail_ShouldRespondWithFormattedJ
             {
                 {"ref", "0test/01"},
                 {"current_arrears_agreement_status", "Breached"},
+                {"current_balance","23.01"},
```
Comment (StevenLeighton21) [inline LBHTenancyAPITest/Test/Controllers/GetAllTenancyDetailsForGivenTenancyRefTest.cs]: need spaces after commas! :D

Comment (LBHRShetty) [reply LBHTenancyAPITest/Test/Controllers/GetAllTenancyDetailsForGivenTenancyRefTest.cs]: fixed it now.

File: LBHTenancyAPI/Controllers/TenanciesController.cs [hunk 2]
```diff
@@ -136,6 +136,7 @@ public async Task<IActionResult> GetTenancyDetails(string tenancyRef)
                     {
                         {"ref", tenancy.TenancyRef},
                         {"current_arrears_agreement_status", tenancy.ArrearsAgreementStatus},
+                        {"current_balance",tenancy.CurrentBalance},
```
Comment (LBHRShetty) [inline LBHTenancyAPI/Controllers/TenanciesController.cs]: fixed the space issue

**Comment by StevenLeighton21** [comment]
I would consider renaming the PR to reference _tests_ not _TDDs_ but otherwise, LGTM (looks _great_ to me)

**Pull request merged**